### PR TITLE
Update to Gradle 7.5

### PR DESCRIPTION
**Pull request by reta**
Signed-off-by: Andriy Redko <andriy.redko@aiven.io>

### Description
Update to Gradle 7.5
 
### Issues Resolved
Closes https://github.com/opensearch-project/k-NN/issues/468
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [X] Commits are signed as per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/k-NN/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,7 +9,7 @@ buildscript {
     ext {
         // build.version_qualifier parameter applies to knn plugin artifacts only. OpenSearch version must be set
         // explicitly as 'opensearch.version' property, for instance opensearch.version=2.0.0-rc1-SNAPSHOT
-        opensearch_version = System.getProperty("opensearch.version", "2.1.0-SNAPSHOT")
+        opensearch_version = System.getProperty("opensearch.version", "2.2.0-SNAPSHOT")
```
Comment (martin-gaievski) [inline build.gradle:12]: I think opensearch version bump should be in a separate PR related to release, @junqiu-lei

Comment (reta) [reply build.gradle:12]: Sure, I can do that, part of https://github.com/opensearch-project/k-NN/issues/467 ?

Comment (martin-gaievski) [reply build.gradle:12]: yes, I think so. I also like to get confirmation from @junqiu-lei, he's working on 2.2.0 release.

Comment (junqiu-lei) [reply build.gradle:12]: Thanks @reta, I can take the version bump PR as part of 2.2 release.

**Comment by junqiu-lei** [comment]
Hi @reta, I just created a [PR](https://github.com/opensearch-project/k-NN/pull/472) to bump gradle to 7.5, I'm going to close this PR, thank you for your contribution!
